packages/ckeditor5-image/ckeditor5-metadata.json

```diff
@@ -7,15 +7,18 @@
 			"docs": "features/images/image-upload/images-inserting.html#inserting-images-via-pasting-url-into-editor",
 			"path": "src/autoimage.js",
 			"requires": [
-				"Image"
+				[
+					"ImageBlock",
+					"ImageInline"
+				]
```

Comment:
@psmyrek, AFAIS it's an undocumented feature now. Could you add a comment there?

Reply:
Added.
I also updated the Features’ HTML output page in the docs